### ISSUE TITLE
ImportError: No module named 'natural_earth'

### ISSUE THREAD

**Issue by dacoex** [mathause/regionmask#10]
Trying: http://regionmask.readthedocs.io/en/stable/_static/notebooks/plotting.html

```
import regionmask
regionmask.__version__

Traceback (most recent call last):

  File "<ipython-input-1-e9f83e33c895>", line 1, in <module>
    import regionmask

  File "/miniconda3/envs/mapping/lib/python3.5/site-packages/regionmask/__init__.py", line 25, in <module>
    from . import defined_regions

  File "/miniconda3/envs/mapping/lib/python3.5/site-packages/regionmask/defined_regions/__init__.py", line 7, in <module>
    from natural_earth import natural_earth

ImportError: No module named 'natural_earth'
```

**Comment by mathause** [comment]
Thanks! For some reason this worked for me. Should be fixed on master, now. Will test it and do a bugfix release when I have a proper computer...

**Comment by mathause** [comment]
Ok, there were several python 3 issues that went undetected because travis was not set up properly... I hope they are all fixed now in v0.3.1.
